lib/blocks.php

```diff
@@ -132,18 +130,18 @@ function gutenberg_reregister_core_block_types() {
 			register_block_type_from_metadata( $block_json_file );
 		}
 
-		foreach ( $block_names as $file => $block_names ) {
+		foreach ( $block_names as $file => $sub_block_names ) {
 			if ( ! file_exists( $blocks_dir . $file ) ) {
 				return;
 			}
 
-			if ( is_string( $block_names ) ) {
-				if ( $registry->is_registered( $block_names ) ) {
-					$registry->unregister( $block_names );
+			if ( is_string( $sub_block_names ) ) {
```

Comment:
Do you think since you're making this change, we should normalize `sub_block_names` into an array and remove the duplication of:
```
if ( $registry->is_registered( $sub_block_names ) ) {
	$registry->unregister( $sub_block_names );
}
gutenberg_register_core_block_styles( $sub_block_names );

```

Reply:
mmm :) I'm lazy :P If you ant to push it go for it :P

Reply:
will push and rebase

Reply:
Thank yo :)